integrations/SD_Lon/sd_importer.py

```diff
@@ -488,6 +490,11 @@ def _create_employees(self, persons, skip_manager=False):
             max_rate = -1
             min_id = 999999
             for employment in employments:
+                job_position_id = employment['Profession']['JobPositionIdentifier']
+                if job_position_id in self.skip_job_functions:
+                    logger.info('Skipping {} due to job_pos_id'.format(employment))
```

Comment:
snart vil vi kunne skrive 
logger.info(f"Skipping {employment} due to job_pos_id"), 
og så napper den selv navnet fra locals()

Reply:
Er det ikke først i python nyere-end-det-vi-har?